src/iblt.cpp

```diff
@@ -117,6 +138,13 @@ void CIblt::resize(size_t _expectedNumEntries)
 
     CIblt::n_hash = OptimalNHash(_expectedNumEntries);
 
+    if (salt > VALS_32 / n_hash)
```

Comment:
use std::numeric_limits<uint32_t>::max()

Reply:
Removed entirely in 4eee6a1.